vllm/sampling_params.py

```diff
@@ -210,6 +216,10 @@ def _verify_args(self) -> None:
         if self.prompt_logprobs is not None and self.prompt_logprobs < 0:
             raise ValueError(f"prompt_logprobs must be non-negative, got "
                              f"{self.prompt_logprobs}.")
+        if len(self.stop) > 0 and self.detokenize is False:
```

Comment:
nit: minor simplification
```suggestion
        if self.stop and not self.detokenize:
```